pages/about-us/newsletter/2013/03/en.md

```diff
@@ -1,14 +1,21 @@
 @title = "March"
 
-h2. We need to talk about Facebook
+## We need to talk about Facebook
 
 We have tech collective allies all over the globe. One of these groups posted a compelling critique of activists using Facebook. While we Riseup birds may agree with some, all, or most of the piece, we think it is a nice conversation starter. So this month, please pass this along to your friends, comrades, allies, collective members, volunteers, staff members - whomever you organize with.
 
-"Original article, in german":http://nadir.org/news/Pl%C3%B6tzlich_plappern_Anna_und_Arthur.html
+Original article, in german http://nadir.org/news/Pl%C3%B6tzlich_plappern_Anna_und_Arthur.html
```

Comment:
This should make "Original article, in german" a link

Reply:
Newsletters are in plaintext, which leads that most of the links in them are just simple calls as this one. When I had to figure it out what should I do with them I just followed the rest of the pages which had no links.